runelite-client/src/main/java/net/runelite/client/RuneLite.java

```diff
@@ -177,9 +178,11 @@ public static void main(String[] args) throws Exception
 		parser.accepts("developer-mode", "Enable developer tools");
 		parser.accepts("debug", "Show extra debugging output");
 		parser.accepts("sessionfile", "Use a specified session file")
```

Comment:
You can just store this as local variable so you can get File from it without casting (see updateMode)